charts/loghouse/templates/clickhouse/clickhouse-configmap.yaml

```diff
@@ -50,7 +50,9 @@ data:
     <yandex>
         <profiles>
             <default>
-                <max_memory_usage>10000000000</max_memory_usage>
+                <max_memory_usage>{{ mul .Values.clickhouse.memlimits 1000000 }}</max_memory_usage>
```

Comment:
Why do you think it is right to multiply by `1000000`, rather than by `1024 * 1024`?